Adyen.docc/Components.md

```diff
@@ -26,3 +26,154 @@ In order to have more flexibility over the checkout flow, you can use our Compon
+We recommend creating a new context for each payment attempt.
+
+```swift
+var context: AdyenContext {
+    let apiContext = APIContext(clientKey: clientKey, environment: Environment.test)
+    return AdyenContext(apiContext: apiContext,
+                        payment: payment,
+                        analyticsConfiguration: analyticsConfiguration)
+}
+```
+
+Create an instance of `AdyenSession.Configuration` with the response you received from the `/sessions` call and the `AdyenContext` instance.
+
+```swift
+let configuration = AdyenSession.Configuration(sessionIdentifier: response.sessionId,
+                                               initialSessionData: response.sessionData,
+                                               context: context)
+```
+
+Call the static `initialize` function of the `AdyenSession` by providing the configuration and the delegates, which will asynchronously create and return the session instance.
+
+```swift
+AdyenSession.initialize(with: configuration, delegate: self, presentationDelegate: self) { [weak self] result in
+    switch result {
+    case let .success(session):
+        // store the session object
+        self?.session = session
+    case let .failure(error):
+        // handle the error
+    }
+}
+```
+
+Fetch `paymentMethod` from `paymentMethods` response or create one manually. 
+
+```swift
+guard let paymentMethods = paymentMethods,
+      let paymentMethod = paymentMethods.paymentMethod(ofType: CardPaymentMethod.self) else { return nil }
+```
+
+You can configure your component by providing `Configuration` object.
+Check specific payment method pages to confirm if you need to include additional required parameters.
+
+```swift
+let style = FormComponentStyle()
+let configuration = CardComponent.Configuration(style: style)
+configuration.showsHolderNameField = true
+return CardComponent(paymentMethod: paymentMethod,
+                     context: context,
+                     configuration: configuration)
+```
+
+Some payment methods need additional configuration. For example `ApplePayComponent`. These payment method specific configuration parameters can be set in an instance of `DropInComponent.Configuration`:
+
+```swift
+let summaryItems = [
+                      PKPaymentSummaryItem(label: "Item A", amount: 75, type: .final),
+                      PKPaymentSummaryItem(label: "Item B", amount: 25, type: .final),
+                      PKPaymentSummaryItem(label: "My Company", amount: 100, type: .final)
+                   ]
+let applePayment = try ApplePayPayment(countryCode: "US",
+                                       currencyCode: "USD",
+                                       summaryItems: summaryItems)
+
+let dropInConfiguration = DropInComponent.Configuration()
+dropInConfiguration.applePay = .init(payment: applePayment,
+                               merchantIdentifier: "merchant.com.adyen.MY_MERCHANT_ID")
```

Comment:
why is dropIn mentioned in a section about individual components?

Reply:
👍